docs/cases/cases-overview.asciidoc

```diff
@@ -118,9 +118,9 @@ IMPORTANT: Before importing Lens visualizations, Timelines, or alerts into a spa
 [float]
 [[cases-export]]
 === Export a case
-Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (NDJSON) file: case details, user actions, text string comments, case alerts, and lens visualizations.
+Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (`.ndjson`) file: case details, user actions, text string comments, case alerts, and lens visualizations (which are exported as JSON blobs).
 
-NOTE: Exported alerts contain metadata that allow the case to reference the appropriate alerts after they're imported. Lens visualizations are exported as JSON blobs.
+IMPORTANT: Alerts attached to a case are exported as references containing alert metadata. When a <<cases-import, case is imported>>, it uses these references to re-connect to the appropriate alerts. However, alert references only resolve if the correct alert documents are available in the environment. If they're not, alert details will not load when you access them from the imported case. Links to the rule that produced the alert will resolve if the alert is present in the environment.
```

Comment:
The first sentence uses "attach" and that seems like a good way to describe the relationship between alerts and cases. Also, "correct" was a little unclear (what makes an alert correct, or incorrect?).
```suggestion
IMPORTANT: Alerts attached to a case are exported as references containing alert metadata. When a <<cases-import, case is imported>>, it uses these references to reattach the appropriate alerts. However, alert references only resolve if the referenced alert documents are available in the environment. If they're not, alert details will not load when you access them from the imported case. Links to the rule that produced the alert will resolve if the alert is present in the environment.
```

Reply:
Agree that "referenced alerts" is less ambiguous than "correct alerts" in this context. I'm not sure whether "reattach" is more suitable than "reconnect" though. @spong from your perspective, are they basically communicating the same idea, or is one more accurate than the other?

Reply:
Perhaps `find`, `locate`, or `look up` would be a suitable replacement for attach/connect here, as there is no technical re-association between the case/alert on import, but rather there's just a broken reference when trying to use features requiring the alert document(s) and it isn't available. You would see the same behavior if the alert document was deleted (and the case was never exported/imported). Does that make sense?

So maybe something along the lines:

```suggestion
IMPORTANT: Alerts attached to a case are exported as references containing alert metadata, so if a <<cases-import, case is imported>> into a cluster without the attached alerts, features relying on the full alert document will not function (e.g. viewing Alert Details, Rule Details, or syncing Alert status with the Case).
```

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/156483087-98785008-6128-404f-a8b4-63e4ad99fe9b.png" />
</p>